README.md

```diff
@@ -1,95 +1,32 @@
-
-* For Mac, Windows, and Linux users wishing to run docker in a VM, start docker machine.  Linux users who
-  have Docker engine (> 1.10) on their local machine can skip this step
-    * `docker-machine create -d virtualbox --virtualbox-memory "2048" apix`
-    * `eval $(docker-machine env apix)`
-    * _Make sure that the api-x code (the fcrepo-api-x) directory is accessible to the docker machine VM_.  For
-    mac users, if the code is in your home sirectory somewhere under /Users, then you're good.  Linux users
-    have some work to do, see [setting up docker machine](docker/README.md)
-* run `mvn clean install -PPOC` to build the api-x code
-* run `docker-compose up -d` to build & run docker images of API-X services.
-
-For more information see [docker/README.md](docker/README.md)
-
-## Verification
-These verification steps can be used to assure that the docker instance(s) are running and are accessible.
-
-First, determine the address `${ADDR}` of the machine running Docker
-* For Vagrant users, this is pre-configured to be `10.11.8.11`
-* For Docker Compose users:
-	* For Linux duning services directly on a local docker, this is `localhost`
-	* For Docker Machine users, you can find this by running `docker-machine ip apix`
-
-Next, verify that some services are running
-* Verify Fedora is running. Point your browser to `http://${ADDR}:8080/fcrepo/rest`
-	* You should see Fedora, with an empty root container
-	* Create an object through Fedora's UI
-* point your browser to Fuseki at `http://${ADDR}:3030/fcrepo-triple-index`
-	* You should see some triples!
-	* Note, ths indexing process is asynchronous, so there may be a delay before these are visible
-* Look at Fedora as seen through a simple reverse proxy service `http://${ADDR}:8082/fcrepo/rest`
+API-X is a framework for binding services to repository objects in order to extend the functionality of a Fedora 4 repository. It provides a data model for expressing how services bind to repository resources, registries that support service discovery, and an HTTP middleware layer that exposes services as endpoints or mediates access to the repository transparently via these services.
+
+For more information, please see the [technical overview](src/site/markdown/apix-design-overview.md) or try out a docker based [demo](https://github.com/fcrepo4-labs/fcrepo-api-x-demo#summary) that highlights what API-X is and does by providing step-by-step evaluation tasks.
+
+## Building
+
+To build API-X, do:
+
+    mvn clean install
+
+## OSGi deployment (Karaf 4.x)
+
+API-X is distributed as OSGi bundles and provides Karaf feature descriptors, making it easy to deploy in a Karaf container. There are several ways to install these features, please refer to the [karaf provisioning documentation](https://karaf.apache.org/manual/latest/#_provisioning) for more information.  
+
+### Manual installation in Karaf
+To do this, first, add the `toolbox-features` repository:
```

Comment:
Should we make it clear that you're to execute these commands from the Karaf console?